handle-errors/go.mod

```diff
@@ -0,0 +1,3 @@
+module jamescurtin/handle-errors
+
+go 1.16
```

Comment:
What do you think about updating the golang image tag in `handle-errors/Dockerfile` such that it is pinned to a 1.16 specifically? That would help us avoid potential situations in the future were there are breaking changes that go un-noticed.

Switching from `golang:alpine` to `golang:1.16-alpine3.13` should do the trick

Reply:
Definitely the way to go.  I’ll include it later tonight.

Reply:
Amending this slightly, so the import path reflects the actual repository here on github.